core/src/main/java/hudson/slaves/NodeProvisioner.java

```diff
@@ -598,13 +598,12 @@ public void recordPendingLaunches(Collection<PlannedNode> plannedNodes) {
          */
         @Override
         public String toString() {
-            final StringBuilder sb = new StringBuilder("StrategyState{");
-            sb.append("label=").append(label);
-            sb.append(", snapshot=").append(snapshot);
-            sb.append(", plannedCapacitySnapshot=").append(plannedCapacitySnapshot);
-            sb.append(", additionalPlannedCapacity=").append(additionalPlannedCapacity);
-            sb.append('}');
-            return sb.toString();
+            String sb = "StrategyState{" + "label=" + label +
+                    ", snapshot=" + snapshot +
+                    ", plannedCapacitySnapshot=" + plannedCapacitySnapshot +
+                    ", additionalPlannedCapacity=" + additionalPlannedCapacity +
+                    '}';
+            return sb;
```

Comment:
Not a big fan of mixing single and double quotes in the same String construction, but it was the case in the previous code, so nothing else to say :)

Reply:
It's an optimization for string concatenation, another IntelliJ code suggestion feature.